src/PensionsRegulatorApi/Controllers/PensionsRegulatorController.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using MediatR;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using PensionsRegulatorApi.Application.Queries;
+using PensionsRegulatorApi.Domain;
+
+namespace PensionsRegulatorApi.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class PensionsRegulatorController : ControllerBase
+    {
+        private readonly IRequestHandler<GetOrganisations, IEnumerable<Organisation>> _getOrganisationsHandler;
+        private readonly ILogger<PensionsRegulatorController> _logger;
+
+        public PensionsRegulatorController(IRequestHandler<GetOrganisations, IEnumerable<Organisation>> getOrganisationsHandler, ILogger<PensionsRegulatorController> logger)
+        {
+            _getOrganisationsHandler = getOrganisationsHandler;
```

Comment:
Is there a reason the we are injecting the handler directly, rather than injection `IMediator` and calling `Send` ?

Reply:
no, I will fix